__tests__/.eslintrc

```diff
@@ -1,6 +1,13 @@
 {
-  "extends": "@readme/eslint-config/testing",
+  "extends": [
+    "plugin:vitest/all"
```

Comment:
Will replace this with `@readme/eslint-config/testing/vitest` when https://github.com/readmeio/standards/pull/723 is merged and published.